android/KMEA/app/src/main/assets/android-host.js

```diff
@@ -308,6 +308,12 @@ function hideKeyboard() {
   window.location.hash = 'hideKeyboard' + fragmentToggle;
 }
 
+function showKeyboard() {
+  var kmw=window['keyman'];
+  // Refresh KMW OSK
+  kmw.correctOSKTextSize();
```

Comment:
There's no value in the indirection of the variable -- we should eliminate elsewhere in the future but at least let's not make it worse 😁 

```suggestion
  // Refresh KMW OSK
  keyman.correctOSKTextSize();
```